Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -0,0 +1,97 @@
+"""This contains the plotting and geo data management methods for the COVIDcast signals.
+
+Shapefiles are sourced from the 2019 US Census Cartographic Boundary Files
+https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html
+Scale is 1:5,000,000
+"""
+
+import geopandas as gpd
+import pandas as pd
+
+import pkg_resources
+from covidcast.covidcast import _detect_metadata
+
+SHAPEFILE_PATHS = {"county": "shapefiles/cb_2019_us_county_5m.zip",
+                   "state": "shapefiles/cb_2019_us_state_5m.zip"}
+
+
+def get_geo_df(data: pd.DataFrame,
+               geo_value_col: str = "geo_value",
+               geo_type_col: str = "geo_type") -> gpd.GeoDataFrame:
+    """Append polygons to a dataframe for a given geography and return a geoDF with this info.
+
+    This method takes in a pandas DataFrame object and returns a GeoDataFrame object from the
+    geopandas package (https://geopandas.org/).
```

Comment:
```suggestion
    `geopandas package <https://geopandas.org/>`__.
```

reStructuredText links are weeeiiird

Reply:
thanks, I haven't worked with sphinx before so will have to read up on it